lib/vehicle.rb

```diff
@@ -22,4 +27,11 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def plate_type
+    @plate_type = if antique? then :antique
```

Comment:
If you remove `@plate_type` from your initialize then this method then your `if/else` logic could look like:
```
if antique?
   :antique
elsif @engine == :ev 
   :ev 
else
   :regular
end
```